File: 9-palindrome-number/9-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        num: int = x
        rev: int = 0
        while num > 0:
            remainder: int = num%10
            rev: int = rev * 10 + remainder
            num = num//10
        return rev == x
            
            
        

File: 9-palindrome-number/9-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        
        long reverse = 0;
        int temp = x;
        int reminder;
        
        while(temp > 0){
            reminder = temp%10;
            reverse = reverse*10 + reminder;
            temp = temp/10;
        }
        if(reverse == x){
            return true;
        }
        return false;
        
    }
};